cluster/common.sh

```diff
@@ -50,6 +50,7 @@ export BUCKET_LOCATION=${BUCKET_LOCATION:-"asia-east1"}
 export CBT_INSTANCE=${CBT_INSTANCE:-"minigo-instance"}
 export CBT_ZONE=${CBT_ZONE:-"us-central1-b"}
 export CBT_TABLE=${CBT_TABLE:-"games"}
+export CBT_EVAL_TABLE=${CBT_EVAL_TABLE:-"eval_games"}
```

Comment:
The two tables right now have the form `v15-games` and `v15-games-nr`. Should this be formed like `v15-games-eval`?

Actually, I believe that we wanted to move to underscores for the table names for some compatibility reasons, and call the non-resign ones "calibration", so maybe the end result should be

  - `v15_games`
  - `v15_games_cal`
  - `v15_games_eval`

Reply:
I think eval is less tightly coupled.

We have maybe ~50-200k rows of data per run + ~10k cross eval + ~5k random things mixed in.

I'm planning to place them all in the same table but with difference key prefix (this remains to be coded).

Ultimately I'd rather write filters on keys than figure out how to join tables.